tests/test_zeus_client.py

```diff
@@ -68,23 +81,22 @@ def test_validate_dates(self):
     @mock.patch('zeus.interfaces.rest.requests')
     def test_post_empty_log(self, mock_requests):
         logs = []
+        url = '{}/logs/{}/ZeusTest'.format(FAKE_SERVER, FAKE_TOKEN)
```

Comment:
I still can not get the point. Meanwhile figured out:

```python
urlparse.urljoin('localhost', posixpath.join('logs', 'token', 'test'))
# => 'logs/token/test'

urlparse.urljoin('localhost:8000', posixpath.join('logs', 'token', 'test'))
# => 'logs/token/test'

urlparse.urljoin('localhost:8000/', posixpath.join('logs', 'token', 'test'))
# => 'logs/token/test'
```

(user needs to set `http`, or `https` or we need to check)

Don't you think just simply do formatting good enough?

```python
url = '{}/logs/{}/test'.format(host, token)

# => 'localhost/logs/token/test'
# => 'localhost:8000/logs/token/test'
# => 'http://localhost:8000/logs/token/test'
```